clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -265,4 +277,13 @@ public UUIDSerde() {
     static public Serde<Void> Void() {
         return new VoidSerde();
     }
+
+    /*
+     * A serde for {@code List} type
+     */
+    public static <L extends List<Inner>, Inner> Serde<List<Inner>>
+        ListSerde(Class<L> listClass, Serde<Inner> innerSerde) {
+        return new ListSerde<>(listClass, innerSerde);
+    }
```

Comment:
This method is hard to read... Can we format it differently? (Maybe a empty line before `return...` is sufficient?)